speech-translation/translator.py

```
@@ -1,10 +1,16 @@
+# pip install -r requirements.txt azure-cognitiveservices-speech==1.42.0
+
+
 from dotenv import load_dotenv
+
 from datetime import datetime
```

Ruff (0.12.2)

`datetime.datetime` imported but unused

Remove unused import: `datetime.datetime`

(F401)